libs/designsystem/src/lib/components/checkbox/checkbox.component.html

```diff
@@ -1,22 +1,10 @@
-<!-- Since there is a bug in Ionic when binding to the `mode` property of ion-checkbox,
-a switch between templates is used -->
-<ng-container *ngIf="shape === 'circle'; else squareBox">
+<div [id]="_labelId">
```

Comment:
Consider removing wrapper div and apply id to an element wrapping `text`...

Reply:
We need a wrapper inside kirby-checkbox, to contain the clickable area around the checkbox and text.

If I remove div. Then the kirby-checkbox gets the position:relative instead, and html label will fill the kirby-checkbox. 
But we want the kirby-checkbox to be display: block to force a linebreak for the next checkbox. This results in the entire line being clickable all the way to the right of the screen. I don't think that is what we want. The clickable area should be over the checkbox and the text until the text stops.

Reply:
Good point 👍